aiida_common_workflows/common/visualization/dissociation.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Utilities to visualize a dissociation curve based on set of distances and energies."""
+
+import typing
+
+import pylab as plt
```

Comment:
I know I probably added this alias, but that is basically because I almost never plot using matplotlib and took it straight from the docs. Reading it here, I don't see why the hell we need this alias though..

Reply:
I guess because it's very common to use:

```Python
import matplotlib.pyplot as plt
```

So `plt` just feels nice and recognisable. That's probably also why it's in the docs?

Looking into this a bit further, it seems that using `pylab` - which is basically a convenience module that includes both `matplotlib.pyplot` and `numpy` in a single namespace - is [no longer recommended](https://matplotlib.org/stable/api/index.html?highlight=pylab#module-pylab). So I'll change it to:

```suggestion
import matplotlib.pyplot as plt
```

Reply:
ok, that makes sense. Happy with that last suggestion